usefulUtilities/portal/README.md

```diff
@@ -0,0 +1,19 @@
+# Portal
+When this script is attached to a zone, the zone serves as a portal, which can be used to transport avatars between domains and within domains.
+
+## Setup
+1. Rez a zone entity that you want to act as the portal.

```

Comment:
remove the period in the end to match the others

Reply:
all set